src/components/layout/Section.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+
+export const VALID_BG_NORMAL = ['white', 'haze', 'lightGray'];
+
+export const VALID_BG_INVERTED = ['navy'];
+
+export const VALID_PADDING = ['default', 'double', 'half', 'none'];
+
+/**
+ * @param {String} axis 'h' or 'v'
+ * @param {String} amount
+ * @returns {String} padding utility classes
+ */
+export const getPaddingClasses = (axis, amount) => {
+  const sides = axis === 'h' ? ['left', 'right'] : ['top', 'bottom'];
+
+  let classes = `padding--${sides[0]} padding--${sides[1]}`;
+  if (amount !== 'default') {
+    classes = `padding--${sides[0]}--${amount} padding--${sides[1]}--${amount}`;
+  }
+
+  return classes;
+};
+
+/**
+ * @param {String} direction
+ * @returns {String} border utility classes
+ */
+export const getBorderClasses = (direction) => {
+  let classes = '';
+  switch (direction) {
+    case 'h':
+      classes = 'border--left border--right';
+      break;
+    case 'v':
+      classes = 'border--top border--bottom';
+      break;
+    default:
+      classes = `border--${direction}`;
+  }
+  return classes;
+};
+
+/**
+ * @param {Object} props react props
+ * @returns {ReactElement}
+ */
+const Section = ({ hPadding, vPadding, bgColor, border, children }) => {
+  const classNames = cx(
+    `bgColor--${bgColor}`,
+    'display--block',
+    getPaddingClasses('h', hPadding),
+    getPaddingClasses('v', vPadding),
+    {
+      inverted: VALID_BG_INVERTED.includes(bgColor),
+      [getBorderClasses(border)]: Boolean(border),
+    }
+  );
+  const styles = {
+    backgroundClip: 'border-box',
+  };
+
+  return (
+    <div style={styles} className={classNames}>
+      {children}
+    </div>
+  );
+};
+
+Section.defaultProps = {
+  hPadding: 'default',
+  vPadding: 'default',
+  bgColor: 'white',
```

Comment:
Would we support transparent background colors with this? (and if so, would white or transparent operate as the default)?

Reply:
I'm thinking that `Section` should always set a background color, as background color is part of its responsibility. If we support a transparent version, the background color is less contained to the section component and relies on parent components to manage backgrounds

Reply:
Hmmm that doesn't entirely make sense to me. So like... you can also set border in Section but it's not required. I also do feel like in a lot of cases, you'll have one white thing (let's just say a `Card`) and assuming design doesn't change color, you're just trying to divide up the sections... so would you force the developer to, in every section inside that white card, redefine it to be white... and it would make it harder to go back and change the color, and it would also make any sort of color change on interaction for the card non-doable, as you'll have a bunch of white boxes on top. 

We should chat about this just so we're aligned and I get where you're coming from.

Reply:
Talked offline. This actually makes a lot more sense to me now, and forces the concern of display to be always on section, rather than fall through, and thereby forces our colors to always be accurate (you can't put a section on green and have it display as green).